### DISCUSSION TITLE
Having trouble adding Deliberate model

### DISCUSSION THREAD

**Discussion by DarkClippy** [vladmandic/sdnext#2579] [Q&A]
I recently downloaded the latest version of Deliberate from HuggingFace, @ https://huggingface.co/XpucT/Deliberate, and I can't figure out how to add this model to SD.Next. Searching for it on the HuggingFace page has no results, and I couldn't seem to find a way of adding it manually.

It's literally downloaded to my computer already.. _**In**_ the models directory, so how can I add it?
![deliberate1](https://github.com/vladmandic/automatic/assets/34817440/9c192d35-d8cc-4bce-b0e4-0bcc47005da3)

**Comment by brknsoul** [comment]
Deliberate looks like a bog-standard Checkpoint model. Those go into `models\Stable-diffusion`

**Comment by ETA-22** [comment]
Think it should be in /models/stable-diffusion not /diffusers
